net/pfSense-pkg-haproxy/files/usr/local/pkg/haproxy/haproxy.inc

```diff
@@ -2157,18 +2180,36 @@ function haproxy_plugin_certificates($pluginparams) {
 				}
 			}
 		}
+		
+		$a_backends = $config['installedpackages']['haproxy']['ha_pools']['item'];
+		if (is_array($a_backends)) {
+			foreach ($a_backends as &$backend) {
```

Comment:
Why use a reference here (&$backend) if this variable is not changed inside the loop?